heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerMain.java

```diff
@@ -115,6 +115,13 @@ private static Options constructOptions() {
         .argName("component parallelism")
         .build();
 
+    Option runtimeConfig = Option.builder("f")
```

Comment:
what about no short name or "rc"? 

Also instead of doing a list you can do something like this
final Option property = Option.builder("rc")
        .argName("comp:property=value")
        .numberOfArgs(2)
        .valueSeparator()
        .desc("use value for given property")
        .build();
then you can pass values like this:
-rc word:prop1=value1 -rc word:prop2=value2

Reply:
sounds good! Didn't even thinking about two characters. :D

Reply:
We do want to keep the current format though to be consistent to component_parallelism.

[comp:]<name>:<value>,[comp:]<name>:<value>,.....

The number of component can be infinity, not really 2.

This is the argument for scheduler, not the CLI argument, which is in python. The CLI argument is in the format in your example.